softvis3d-frontend/src/classes/status/LoadAction.ts

```diff
@@ -14,16 +14,32 @@ export default class LoadAction extends StatusAction {
         this._current = current;
     }
 
-    public get limit(): number {
+    public get limit(): number | undefined {
```

Comment:
The thing is that the limit can be undefined. There are LoadAction which defined a limit and ones which did not. In addition, the limit can be changed via "incrementMax". This is needed by the asynchronous recursive logic where it is not clear in the beginning how many requests it has to do.

But you're right, the resolve of undefined in all the classes is not so nice. I created a method hasLimits() as follows:

`public hasLimit(): boolean {
        return typeof this.current !== "undefined" && typeof this.limit !== "undefined";
}`

Unfortunately, the compile creates an undefined exception also the above method was called:

`if (this.hasLimit()) {
            this._limit = this._limit + 1;
 }`

"Object is possibly 'undefined'." Any ideas how to solve this?